pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -364,19 +364,24 @@ func (f *Fit) isSchedulableAfterPodScaleDown(targetPod, originalPod, modifiedPod
 // isSchedulableAfterNodeChange is invoked whenever a node added or changed. It checks whether
 // that change made a previously unschedulable pod schedulable.
 func (f *Fit) isSchedulableAfterNodeChange(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
-	_, modifiedNode, err := schedutil.As[*v1.Node](oldObj, newObj)
+	originalNode, modifiedNode, err := schedutil.As[*v1.Node](oldObj, newObj)
 	if err != nil {
 		return framework.Queue, err
 	}
-	// TODO: also check if the original node meets the pod's resource requestments once preCheck is completely removed.
```

Comment:
Can we simply use annotations to describe the re-queue scenario, so that the code is easier to understand? :)

Reply:
Added comments to all these cases, so PTAL